src/components/static/prelive/Prelive.jsx

```diff
@@ -6,12 +6,19 @@ import Schedule from "./schedule/Schedule";
 
 const Prelive = () => {
   return (
-    <div className="flex flex-col w-full items-center overflow-y-scroll gap-10">
+    <div className="flex flex-col w-full items-center overflow-y-scroll">
       <Landing />
-      <About />
-      <Schedule />
+      <div className="flex flex-row justify-center w-full">
```

Comment:
div should be implemented directly into the components

Reply:
I realized that I actually don't need that div, so I'll delete the one at line 11. 
But for all other divs with pt-10, I feel like they should be implemented outside the components, because the pt-10 part  is specific to only this situation. If the component were to be used in other situations I feel that having pt-10 could be a hinderance.
The reason I needed pt-10 for each div is because there was originally a gap-10 that applied between every single component, and I didn't want the gap to apply between everything, I want <Landing/> and <About/> to not have that gap-10 or else there'd be the awkward empty space.